src/utils/helpers/events/events.js

```diff
@@ -367,6 +367,42 @@ const Events = {
       composedPath(ev)
     );
   },
+
+  /**
+   * Determines if the key pressed is an accent key
+   *
+   * @method isAccentKey
+   * @param {Event} ev A JavaScript event
+   * @returns {Boolean} true if end key
+   * */
+  isAccentKey: (ev) => {
```

Comment:
These functions should accept an Event, this implementation is expecting a string. However, we should be able to safely delete this method.